### PR TITLE
Add grapql mutation 

### LINKED ISSUE #35: Enable graphql mutation
Currently the server does not have a graphql mutation query

## Todo

Allow to create `department` via `mutation`

### PR DESCRIPTION
**Pull request by sushie1984**
Allows to create `department` via graphql mutation functionality

### Where to start reviewing
[`create_department.rb`](https://github.com/sushie1984/rails-graphql-server/pull/31/files#diff-16877d68c4b1c9342c487b55d51c109d)

### Manual testing
* Start service
* With your favorite tool perform `mutation` queries against the server
* If succeed expect a json response of the created `department`
* If failed expect a json response with the errors informing about why it failed

### References
[Project Ticket](https://github.com/sushie1984/rails-graphql-server/projects/1#card-36543420)

### CODE REVIEW

File: app/graphql/input_objects/department_attributes.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module InputObjects
+  class DepartmentAttributes < Types::BaseInputObject
+    description 'Attributes for modifying a department entry'
+    argument :name, String, 'The name of the department', required: true
+    argument :location, String, 'The location of the deparment', required: true
```
Comment (julia-wo) [inline app/graphql/input_objects/department_attributes.rb:7]: 'The location of the deparment' -> 'The location of the department'

File: spec/graphql/mutations/create_department_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    end
+
+    it 'responds with no errors' do
+      is_expected.to include(errors: [])
+    end
+
+    context 'when department cannot be created' do
+      let(:attributes) { { name: nil, location: nil } }
+
+      it 'responds with no created deparment' do
```
Comment (julia-wo) [inline spec/graphql/mutations/create_department_spec.rb:28]: Nit - typo: 'responds with no created deparment' -> 'responds with no created **department**'

File: app/graphql/input_objects/department_attributes.rb (hunk 1 above)
Comment (sushie1984) [reply app/graphql/input_objects/department_attributes.rb]: Done

File: spec/graphql/mutations/create_department_spec.rb (hunk 2 above)
Comment (sushie1984) [reply spec/graphql/mutations/create_department_spec.rb]: Done

**Pull request merged**